docs/guides/monitor/process.md

```diff
@@ -0,0 +1,297 @@
+<!--
+title: Monitor any process in real-time with Netdata
+description: "Tap into Netdata's powerful collectors, with per-second utilization metrics for every process, to troubleshoot faster and make data-informed decisions."
+image: /img/seo/guides/monitor/process.png
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/guides/monitor/process.md
+-->
+
+# Monitor any process in real-time with Netdata
+
+Netdata is more than a heaping of generic system-level metrics and visualizations. Instead of providing only a bird's
```

Comment:
I'm not sure 'heaping' can be used as a noun? It was a bit confusing/awkward for me.